packages/twenty-server/src/engine/core-modules/workspace/dtos/update-workspace-input.ts

```diff
@@ -13,6 +15,7 @@ export class UpdateWorkspaceInput {
   @IsString()
   @IsOptional()
   @Matches(/^[a-z0-9][a-z0-9-]{1,28}[a-z0-9]$/)
+  @ForbiddenWords(['demo'])
```

Comment:
LGTM but I feel this validation should happen at the model level not in the dto which is the graphql input (we could actually justify that they happen in both points)